Makefile

```diff
@@ -81,12 +81,15 @@ codegen: ## Generate code. Must be run if changes are made to ./pkg/apis/...
 release: ## Generate release manifests and publish a versioned container image.
 	$(WITH_GOFLAGS) ./hack/release.sh
 
-nightly: toolchain## Generate nightly release manifests and publish a versioned container image.
-	$(WITH_GOFLAGS) ./hack/nightly.sh
+nightly: ## Tag the latest snapshot release with timestamp
+	./hack/add-snapshot-tag.sh $(shell git rev-parse HEAD) $(shell date +"%Y%m%d")
 
-snapshot: ## Generate nightly release manifests and publish a versioned container image.
+snapshot: ## Generate a snapshot release out of the current commit
 	$(WITH_GOFLAGS) ./hack/snapshot.sh
 
+stablerelease: ## Tags the snapshot release of the current commit with the latest tag available, for prod launch
+	./hack/add-snapshot-tag.sh $(shell git rev-parse HEAD) $(shell git tag | sort -V | tail -1)
```

Comment:
can we use `git describe --tags --always`. That command will make sure HEAD is on the current release tag and if not append the commit hash.  It might be worth putting a VERSION env var at the top of the Makefile to replace this command and the linker flags that sets the version on line 5.

Reply:
good idea, switched to `git describe --tags --exact-match` as discussed offline.